lib/CirrusSearchRequest.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const nodemw = require( './nodemw' );
+
+module.exports = class CirrusSearchRequest {
+
+	constructor( resultColumnName, property, value ) {
+		this.resultColumnName = resultColumnName;
+		this.property = property;
+		this.value = value;
+	}
+
+	respond( request, response ) {
+		nodemw.search( `haswbstatement:${this.property}=${this.value}`, ( _err, searchResults ) => {
```

Comment:
This won’t find lexemes (since the Lexeme: namespace isn’t searched by default), but apparently nodemw doesn’t let you adjust the search namespaces without switching from `search` to a more generic method :/

Reply:
Done! I set `srnamespace` to `*` to search all namespaces. Ideally we would probably want to search only entity namespaces, but I think this is ok for now.